sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -645,6 +645,18 @@ def test_issue_8866():
     assert expand_log(log(l1, b1)) == expand_log(log(l2, b2))
 
 
+def test_log_expand_factor():
+    assert (log(18)/log(3) - 2).expand(factor=True) == log(2)/log(3)
+    assert (log(12)/log(2)).expand(factor=True) == log(3)/log(2) + 2
+    assert (log(15)/log(3)).expand(factor=True) == 1 + log(5)/log(3)
+    assert (log(2)/(-log(12) + log(24))).expand(factor=True) == 1
+
+    assert expand_log(log(12), factor=True) == log(3) + 2*log(2)
+    assert expand_log(log(21)/log(7), factor=False) == (log(3) + log(7))/log(7)
+    assert expand_log(log(45)/log(5) + log(20) + log(4), factor=False) == (log(5) + 2*log(3))/log(5) + log(20) + 2*log(2)
```

Comment:
This line is longer than 79 characters.
Can you split it into two lines?

Check [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) for reference.

Reply:
I have made the changes, please check.